db/migrate/20250130101122_add_organization_id_to_credit_notes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddOrganizationIdToCreditNotes < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :credit_notes, :organization, type: :uuid, index: {algorithm: :concurrently}
+
+    add_foreign_key :credit_notes, :organizations, validate: false
```

Comment:
I think every statement needs to be its own migration.

Reply:
okay, not sure about this, strong migrations has not complained so far.... why it needs to be in various migrations?

Reply:
the reason would be to limit the amount of statements that are executed without transaction. I don't think it matters a lot, but I believe it's good practice to split the migration into multiple files :)